### ISSUE TITLE
Dashboard loose ends

### ISSUE THREAD

**Issue by mariusandra** [PostHog/posthog#752]
There are still some things to fix with the dashboards:

- [x] Dragging from a link still causes problems: ![screencast 2020-05-13 14-18-06](https://user-images.githubusercontent.com/53387/81811240-9c306500-9524-11ea-909c-f18ddfd0c934.gif)

- [x] It's rather annoying to scroll on a touch device. It just keeps moving the dashboard elements around without actually scrolling the page. Perhaps there should be some default-disabled [EDIT] button that enables moving panels around? If so, should it be there only for touch or also for other devices? Alternatively, we could try starting to move the panels on touch devices only after you hold down your finger for a few seconds.

- [x] It's quite impossible to resize dashboards on touch devices. The resize handler is just too small.

- [x] Editing dashboard items #712 would be great to have.

- [x] I'm starting to find it annoying that if I click "dashboards" in the menu, I get an "index" page with just one link "Default Dashboard". It should probably open the first pinned dashboard by default?

- [x] Adding new items to dashboards puts them in random locations. They should probably all go to the end of the list.

- [x] Can't scroll tables in dashboards if they have a lot of items

- [ ] Undo and View Changes - it's way too easy to accidentally change something and not realise

Nice to haves:

- [x] Duplicate panel button
- [x] Move panel to another dashboard
- [x] Copy panel to another dashboard
- [ ] Can't resize from every corner

Even nicer to haves:

- [x] Color generators should longer, otherwise pie charts, etc get very "white" after 10+ items.
- [ ] Add a global date filter that updates all graphs at the same time.
- [ ] Change funnels JS library - the current one needs to be 100% redrawn every frame when resizing

**Comment by Twixes** [comment]
These are mostly tied up now. If we find a need for more improvements, let's create some more granular issues.
